run.py

```diff
@@ -20,11 +20,12 @@
         app.config.from_object('config')
         api_init()
         port = int(args.get("--port"))
-
+        print('listen on port %s' % (port))
```

Comment:
* `Listening on port %s` (wording, sentences start with upper case)
* `str.format()` should be prefered over the old `%` formatting
* Superfluous parenthesis (IDE warning)
* Formatting (IDE warnings)